.github/workflows/build-nginx-proxy-image.yml

```diff
@@ -0,0 +1,174 @@
+name: Build Nginx Proxy Docker Image
+
+env:
```

Comment:
I suspect we don't need these env vars, we had a few PRs with these in and have been able to remove them in all cases so far